src/main/java/db/FileReader.java

```diff
@@ -0,0 +1,7 @@
+package db;

```

Comment:
```suggestion
package service;
```
помести все сервисы в пакет service